src/components/Navbar/Navbar.cy.tsx

```diff
@@ -0,0 +1,89 @@
+import Navbar, { NavbarItem } from "./Navbar";
+import {
+  faClock,
+  faCompass,
+  faFutbol,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
+
+const initialItems = [
+  {
+    icon: faClock,
+    title: "Time",
+  },
+  {
+    icon: faFutbol,
+    title: "Comments",
+  },
+  {
+    icon: faCompass,
+    title: "Compass",
+  },
+  {
+    icon: faUser,
+    title: "User",
+  },
+];
+
+describe("<Navbar />", () => {
+  it("renders navbar with 4 items", () => {
+    cy.mount(
+      <Navbar label="Navbar">
+        {initialItems.map((item, i) => (
+          <NavbarItem
+            key={`${item.title}-${i}`}
+            icon={item.icon}
+            iconStyles={{ fontSize: "30px" }}
+            title={item.title}
+            onSelect={() => {}}
+          />
+        ))}
+      </Navbar>
+    );
+  });
+
+  it("should set first item as default active", () => {
```

Comment:
todo: In this test you assert the `defaultActive`, but in the previous test you didn't test that there's no active item. When you have these "conditional" states, make sure to test both scenarios. In other words, the first test should say "renders navbar with 4 items, all non active"

Reply:
nicee